release-tasks.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+STATEMENT_TIMEOUT=180000 bundle exec rails db:migrate
+rails runner "puts 'app load success'"
```

Comment:
I think these tasks need to be joined by `&&`, the reason is because if migrate fails, the script will run rails runner anyway and we don't want that. By joining them with a `&&` we make sure that the second runs only if the first one runs.

Another reason to join them is to have a reliable exit code, which I suspect is what Heroku uses to check if the script has worked or not.

If the migration fails but the runner script succeeds the exit code will be `0` and the deployment will likely move on even if it shouldn't.

You can test this by replacing the script content with:

```shell
foobar
ls | wc -l
```

as you can see the output exit code is 0:

```shell
➜  devto git:(mstruve/pre-release-phase-script) ✗ sh release-tasks.sh
release-tasks.sh: line 6: foobar: command not found
      42
➜  devto git:(mstruve/pre-release-phase-script) ✗ echo $?
0
```

If, instead, I join the two commands:

```shell
foobar && ls | wc -l
```

I have the corect behavior:

```shell
➜  devto git:(mstruve/pre-release-phase-script) ✗ sh release-tasks.sh
release-tasks.sh: line 6: foobar: command not found
➜  devto git:(mstruve/pre-release-phase-script) ✗ echo $?
127
```

now the exit code represents an error and `ls` never ran

Reply:
GREAT catch @rhymes!!!